From: Antonio Radici <antonio@debian.org>
Date: Thu, 27 Feb 2014 16:20:58 +0100
Subject: assumed_charset-compat

On upstream inclusion of the assumed_charset patch (past 1.5.14),
file_charset got renamed to attach_charset. We add it back
for compatibility.

Gbp-Pq: Topic debian-specific
---
 init.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/init.h b/init.h
index 7d319ad..875cad8 100644
--- a/init.h
+++ b/init.h
@@ -211,6 +211,9 @@ struct option_t MuttVars[] = {
   ** Note: for Japanese users, ``iso-2022-*'' must be put at the head
   ** of the value as shown above if included.
   */
+  { "file_charset",	DT_SYN,  R_NONE, UL "attach_charset", 0 },
+  /*
+  */
   { "attach_format",	DT_STR,  R_NONE, UL &AttachFormat, UL "%u%D%I %t%4n %T%.40d%> [%.7m/%.10M, %.6e%?C?, %C?, %s] " },
   /*
   ** .pp
